### DISCUSSION TITLE
phenoage0 calculation

### DISCUSSION THREAD

**Discussion by ramyar1993** [dayoonkwon/BioAge#7] [Q&A] [answered]
Hello,

Thanks for creating this very useful package! I have a question about calculation of the phenotypic age based on the original coefficients. 

This is the formula I saw in the original paper for calculating phenotypic age:
![image](https://user-images.githubusercontent.com/34075343/136076433-f85d91c8-f9be-43f5-9f71-623a70886974.png)

However, I saw an additional step in your calculations for calculating m_orig:
![image](https://user-images.githubusercontent.com/34075343/136076512-82b4991d-7e5a-4d9c-a9d2-a29529463e1a.png)

Could you please explain why there is an additional step? Thanks again!

**Comment by dayoonkwon** [comment, marked as answer]
Hello,

As you can see from Levine 2018 (supplementary methods), the formula you attached is just a part of the equation. In order to fully solve the equation to get PhenoAge, you need to compute a mortality prediction score (CDF(120,x)) first. Then the mortality prediction score is covered into a biological age value (CDF._univariate_(120,age)). Finally, you can solve the equation CDF(120,x) = CDF._univariate_(120,age). 

I hope this helps. Please send me an email if it is not clear.

Best,
Dayoon
